### ISSUE TITLE
How can i train model from scratch

### ISSUE THREAD

**Issue by Hwang-dae-won** [facebookresearch/Detectron#53]
Hello.

I want to train mask rcnn from scratch(not using the pre-trained weight)

I hope weight parameter starts from random initialization.

How can i do this?

**Comment by KaimingHe** [comment]
Training from scratch is possible in terms of coding and can be done without much modification in this code. However, there may be convergence problems, e.g., caused by not using BN, or using BN but with a small mini-batch size. We encourage more research to be done on this.

**Comment by rbgirshick** [comment]
One caveat to add: we noticed just before release that there is currently a bug that will cause a crash when trying to train from scratch (the scale and bias parameters of the AffineChannel ops will not be initialized). We have a patch for this that will hopefully roll out this week. Once that is fixed, leaving `TRAIN.WEIGHTS` as the empty string will trigger training from scratch. As @KaimingHe says, more research needs to be done before one should expect to get good results.

**Comment by rbgirshick** [comment]
Since e59c30bb1a6ced1a310b72d563bd9a60aba84999 was committed, it is now possible to train from scratch by setting `TRAIN.WEIGHTS` to the empty string (equiv. delete from your yaml file). But I want to reinforce Kaiming's point that significant experimentation will be needed to get reasonable results from doing so.
